blocks/blog-home/blog-home.js

```diff
@@ -5,6 +5,115 @@ import {
 const NUM_CARDS_SHOWN_AT_A_TIME = 6;
 let loadMoreElement;
 const MODE = 'blog-home';
+let url = new URL(window.location.href);
+const params = new URLSearchParams(url.search);
+export const selectedAudience = params.get('audience') ? params.get('audience').split(',') : [];
+export const selectedTopics = params.get('topics') ? params.get('topics').split(',') : [];
+export const selectedContentTypes = params.get('content-types') ? params.get('content-types').split(',') : [];
+
+function getSetParam() {
+  if (selectedAudience.length > 0) {
+    params.set('audience', selectedAudience);
+  } else {
+    params.delete('audience');
+  }
+  if (selectedTopics.length > 0) {
+    params.set('topics', selectedTopics);
+  } else {
+    params.delete('topics');
+  }
+  if (selectedContentTypes.length > 0) {
+    params.set('content-types', selectedContentTypes);
+  } else {
+    params.delete('content-types');
+  }
+
+  // Update the URL with the modified parameters
+  url.search = params?.toString();
+  const newURL = url?.toString();
+
+  // Update the browsers history with new URL
+  window.history.replaceState({ path: newURL }, '', newURL);
+}
+
+function pushValueToQueryParameter(group, value) {
+  let audienceIndex;
+  let topicIndex;
+  let contentTypeIndex;
+
+  switch (group) {
+    case 'audience':
+      audienceIndex = selectedAudience?.indexOf(value);
+      if (audienceIndex === -1) {
+        selectedAudience.push(value);
+      }
+      break;
+    case 'topics':
+      topicIndex = selectedTopics.indexOf(value);
+      if (topicIndex === -1) {
+        selectedTopics.push(value);
+      }
+      break;
+    case 'content-types':
+      contentTypeIndex = selectedContentTypes.indexOf(value);
+      if (contentTypeIndex === -1) {
+        selectedContentTypes.push(value);
+      }
+      break;
+    default:
+      console.log('Unknown Type');
```

Comment:
Let's remove this console log check as well as other places where they're being used (eg. line 93).